docs/agents/sematext-agent/releasenotes.md

```diff
@@ -35,17 +35,72 @@ NULL
 
 -->
 
+# Version 3.0.0
+
+Date: July 25, 2023
+
+## New Features
+
+- [**Kubernetes Components**](https://kubernetes.io/docs/concepts/overview/components/) monitoring integration: We are excited to announce that Sematext Agent now supports monitoring of critical Kubernetes components, including **API Server**, **CoreDNS**, **kube-proxy**, **kubelet**, **Scheduler**, and **Etcd**. Gain deeper insights into your Kubernetes cluster's performance with this powerful integration. Check out [Kubernetes Monitoring Integration](https://sematext.com/docs/integration/kubernetes/) for the details.
+- Varnish Cache logs parsing support: With this update, Sematext Agent can now parse fields from your Varnish Cache log messages. Stay tuned for the upcoming release of Varnish Cache logs integration in Sematext Cloud, enhancing your logging experience further.
+
+## Improvements
+
+- Improved discovery for OpenSearch integration: We have enhanced the OpenSearch integration by preventing the discovery of `.json` files, which are not supported. Now, Sematext Agent will only discover and support `.log` files for OpenSearch integration, ensuring a smoother experience.
+
+## Bug Fixes
+
+- SC-16613 Fixed incorrect `Not Configured` status for the log shipper: We have resolved an issue that was causing the log shipper to display an incorrect status message. With this fix, you will now receive accurate log shipper status updates.
```

Comment:
an issue ==> the issue
With this fix, you will now receive accurate log shipper status updates. ==> not necessary